### ISSUE TITLE
Adding secrets on deployment

### ISSUE THREAD

**Issue by vonNiklasson** [google-github-actions/deploy-cloudrun#63]
### TL;DR
It's possible to reference secrets either by mounting them as files or add them as environment variables on deployment (see references). It'd be great to have the option to add them on deployment through the GitHub action as well.

### Design

**Action YAML**
Probably the same way you reference environment variables through deployment today, but with the revision tag as suffix. Preferably the underlying `--update-secrets` tag would be used, so it matches the behaviour of the existing `env_vars` functionality.
```yaml
- secrets = LOCAL_ENV_NAME=secret-name:[revision|latest],...
```
Using a path instead of environment name would mount it as file instead.

**Resources**
- [Link to CLI documentation](https://cloud.google.com/sdk/gcloud/reference/run/deploy#--update-secrets)
- [Link to Cloud Blog](https://cloudblog.withgoogle.com/products/serverless/improving-the-security-of-your-cloud-run-environment/amp/)
- [Link to YouTube demonstration](https://youtu.be/JIE89dneaGo)

**Additional information**
N/A
